index.html

```diff
@@ -29,13 +29,13 @@
     <nav class="full-menu">
       <ul>

```

Comment:
- [ ] OPTIONAL: I would recommend to debug the whole project and make it responsive. I have noticed that you fixed width everywhere in mobile view to `375px` this is introducing a number of issues. For instance, mobile screens can have different width, from `320px` to `425px`, but your  application renders properly only for certain width which is a wrong approach. Most of the time users will have to scroll the body left and right to see the content.

https://user-images.githubusercontent.com/77038610/161718639-c25db573-4b5e-4aa6-abd4-e51f4547f374.mp4

- [ ] REQUIRED: Please make sure that the content of the mobile navbar fproperly fixes the screen, so that users were able to see the content without moving the page left and right. Currently I have to move the page to see the content on the right.

Reply:
- [ ] [Optional][Highly recommended] you fixed the side-scrolling for the mobile version only for the popup menu but the page itself still has the same problem so please fix that.
[Hint]:  You can fix that by adding `box-sizing: border-box;` for the hall `body` and giving all containers of different sections percentage width values instead of fixed ones.